README.md

```diff
@@ -101,14 +105,19 @@ Run through the set up instructions; once complete you will need to run:
 
 **theguardian.com frontend repo**: [https://github.com/guardian/frontend](https://github.com/guardian/frontend)
 
-Run through the set up instructions - note you need to make sure that your Frontend is set up to point at your _local_ Identity, not the `CODE` Identity, which means adding this to your `frontend.properties`:
+Run through the set up instructions - download `frontend` and make sure that your `frontend` project is set up to point at your _local_ Identity, not the `CODE` Identity, which means adding this to your `frontend.properties`:
+
+ `$ cd ~/.gu`
```

Comment:
`$` sign is not a part of the command and may differ on different platforms, I would not use it